spec/models/lettings_log_spec.rb

```diff
@@ -1497,6 +1520,23 @@ def previous_postcode_fields
           end
 
           context "and it is a 23/24 form" do
+            before do
+              Timecop.freeze(Time.zone.local(2023, 5, 1))
+            end
+
+            after do
+              Timecop.unfreeze
+            end
+
+            around do |example|
+              Timecop.freeze(Time.zone.local(2023, 5, 1)) do
+                Singleton.__init__(FormHandler)
+                example.run
+              end
+              Timecop.return
```

Comment:
here i don't think toy will need the `before` and `after` as you've covered this in the `around`. also this line should be able to go? as the `freeze` should only happen to anything with the block

Reply:
These are actually required, it's not ideal but the around wraps around global before and afters which is required for initialising logs that are reused in different time periods.